sw/device/examples/hello_usbdev/BUILD

```diff
@@ -2,10 +2,12 @@
 # Licensed under the Apache License, Version 2.0, see LICENSE for details.
 # SPDX-License-Identifier: Apache-2.0
 
-load("//rules:opentitan.bzl", "OPENTITAN_CPU", "opentitan_flash_binary")
+load("//rules/opentitan:defs.bzl", "opentitan_binary")
+load("//rules:opentitan.bzl", "OPENTITAN_CPU")
 
-opentitan_flash_binary(
+opentitan_binary(
     name = "hello_usbdev",
+    testonly = True,
```

Comment:
If we decide that the example binaries should not be `testonly=True`, we can use `sim_dv_base` and `sim_verilator_base` as the exec_envs.

Reply:
I feel they should be marked as `testonly=True` given that they should only be used as examples.